### PR TITLE
add description and keywords to package.json

### PR DESCRIPTION
**Pull request by juliannzhou**
This PR adds description and keywords to package.json. The description is copied from README and keywords provided by product.

J=SLAP-2190
TEST=manual

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,6 +27,7 @@
     "build:cjs": "tsc -p tsconfig.cjs.json",
     "prepublishOnly": "npm run build",
     "dev": "tsc --watch -p tsconfig.esm.json",
+    "lint": "eslint .",
```
Comment (oshi97) [inline package.json:30]: is this from a separate PR?

File: package.json [hunk 2]
```diff
@@ -1,8 +1,18 @@
...
   "main": "./lib/esm/src/index.js",
   "types": "./lib/esm/src/index.d.ts",
+  "keywords": [
+    "search",
+    "nlp",
+    "ai",
+    "answers",
+    "yext",
+    "react",
+    "components"
```
Comment (nmanu1) [inline package.json:14]: did product want "components" as a keyword here even though this library doesn't provide any components?

Comment (oshi97) [reply package.json:14]: This library technically has a component (AnswersHeadlessProvider) though it's really not about components

**Comment by oshi97** [comment]
let's merge into develop for this one too

File: package.json (hunk 2 above)
Comment (juliannzhou) [reply package.json:14]: Product only gave me one list, so I updated the repos with the same list. Should I ask product to give a list for every public repository?

Comment (nmanu1) [reply package.json:14]: Yeah, I would ask for a list for each repo since they have different purposes

**Review by oshi97 — APPROVED** [review]

**Pull request merged**